src/base/static/libs/maps/mapboxgl-provider.js

```diff
@@ -23,16 +23,54 @@ export default (container, options) => {
     options.control.position,
   );
 
+  const _appendFilters = (existingFilters, ...filtersToAdd) => {
```

Comment:
I'm not too confident about the array manipulation in this method-- am I missing a more functional approach?